geniusyield-market-maker/src/GeniusYield/MarketMaker/Strategies.hs

```diff
@@ -185,6 +188,25 @@ filterOwnOrders mmts users allOrders =
     fromJust
       $ find ((==) poiOwnerKey . toPubKeyHash . pkhUser) users
 
+withPriceEstimate :: (Price -> IO UserActions) -> Strategy
+withPriceEstimate k pp _ mmt = do
+  let mmTokenPair = mkMMTokenPair mmtLovelace mmt
+      (Connection _ providers, _) = orderBookPP pp
+
+  pe <- priceEstimate pp mmTokenPair
+  case pe of
+    PriceSourceFail p -> do
+      logWarn providers $ "One prices provider failed"  -- TODO: give details of failing provider
```

Comment:
Do you plan to tackle this todo in this PR?

Reply:
A warning log has been made for this case but not for others which seem inconsistent.

Reply:
Done.  Tackled this issue in [this commit](https://github.com/geniusyield/market-maker/pull/73/commits/6e78e20a3ffbb434a399bd58c7f5307a5e8d31d7).